test/commands/logcat.js

```diff
@@ -0,0 +1,15 @@
+var fs = require('fs');
+var path = require('path');
+
+/**
+ * Stream sample logcat data to stdout.
+ */
+module.exports = function logcat() {
+  fs
+    .createReadStream(path.resolve(`${__dirname}/../output/logcat`))
+    .pipe(process.stdout);
+
+  setInterval(() => {
+    console.log('V/WLAN_PSA(  209): NL MSG, len[048], NL type[0x11] WNI type[0x5050] len[028]');
```

Comment:
Haha, this is an actual logcat, but in the future when we want to test other logcat integrations we will need this more proper logcat (I think).